test/regression/cases/quality_gate_idle_all_features/experiment.yaml

```diff
@@ -0,0 +1,52 @@
+# Agent 'all features enabled' idle experiment. Represents an agent install with
+# all sub-agents enabled in configuration and no active workload.
```

Comment:
What do you think about adding a blurb about quality gates and a link to our docs in these?

Reply:
Yes, I plan to add this once the corresponding confluence docs exist

Reply:
👀 🙈